modules/core/src/shaderlib/project/viewport-uniforms.js

```diff
@@ -54,8 +57,20 @@ function calculateMatrixAndOffset({
   let {viewProjectionMatrix} = viewport;
 
   let projectionCenter;
+  let shaderCoordinateSystem = coordinateSystem;
+  let shaderCoordinateOrigin = coordinateOrigin;
+
+  if (coordinateSystem === COORDINATE_SYSTEM.LNGLAT_EXPERIMENTAL) {
+    if (coordinateZoom < LNGLAT_EXPERIMENTAL_ZOOM_THRESHOLD) {
+      shaderCoordinateSystem = COORDINATE_SYSTEM.LNGLAT;
```

Comment:
We'll need to define a separate enum for the shader once we make this the default mode. Maybe do that now?

Reply:
Next PR.